docs/research/power-consumption-tests.md

```diff
@@ -0,0 +1,160 @@
+# PC Engines apu3 power consumption tests
+
+We have started research and measurements of power consumption on apu platforms.
+Our goal is to optimize (if possible) power management on coreboot level. First
+stage of work is to collect data from various tests and create reports. For that
+purpose we have used RTE with power monitor sensor.
```

Comment:
@piotr-kleins please link to RTE, what it is and where to get it

Reply:
@pietrushnic Added everything and link to 3mdeb shop.